NAMESPACE

```diff
@@ -1,5 +1,11 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(epidist,epidist_ltcad)
+S3method(epidist_family,epidist_ltcad)
+S3method(epidist_formula,epidist_ltcad)
+S3method(epidist_prepare,default)
```

Comment:
Why is this the only one with a default method? Can we try and get away with a default method for `epidist` that actually does something? For the others can likely be nothing right now or it could just pass through to `brms`

Reply:
(by nothing I usually mean a warning saying this class doesn't have x method implemented - this can really help when people are making their own methods)

Reply:
Why -> I copied `idbrms`.

Yes I think we should have default method for `epidist` see #80.

And yes seems reasonable to add default method for non-implemented classes giving a warning. Will add issue.

Reply:
Added issue  #82.